### PR TITLE
fix broken links

### PR DESCRIPTION
**Pull request by oliver-sanders**
**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.

### CODE REVIEW

**Comment by oliver-sanders** [comment]
(might as well leave the reference to atom up since some people are still using it)

File: src/user-guide/writing-workflows/configuration.rst [hunk 1]
```diff
@@ -65,10 +65,9 @@ Cylc provides syntax plugins for the following editors:
 
 .. _Cylc.tmbundle: https://github.com/cylc/Cylc.tmbundle
 .. _vscode-cylc: https://marketplace.visualstudio.com/items?itemName=cylc.vscode-cylc
-.. _language-cylc: https://atom.io/packages/language-cylc
 
 Atom
-   install the `language-cylc`_ extension.
+   install the "language-cylc" extension.
```
Comment (MetRonnie) [inline src/user-guide/writing-workflows/configuration.rst:70]: atom.io is back online. Or can change this link to https://github.com/cylc/language-cylc if that's more likely to persist into the future

Comment (oliver-sanders) [reply src/user-guide/writing-workflows/configuration.rst]: Done.

File: src/user-guide/writing-workflows/configuration.rst [hunk 2]
```diff
@@ -63,12 +63,12 @@ Syntax Highlighting For Workflow Configuration
...
 Cylc provides syntax plugins for the following editors:
 
+.. _language-cylc: https://github.com/cylc/language-cylc
 .. _Cylc.tmbundle: https://github.com/cylc/Cylc.tmbundle
 .. _vscode-cylc: https://marketplace.visualstudio.com/items?itemName=cylc.vscode-cylc
-.. _language-cylc: https://atom.io/packages/language-cylc
 
 Atom
-   install the `language-cylc`_ extension.
+   install the `language-cylc` extension.
```
Comment (MetRonnie) [inline src/user-guide/writing-workflows/configuration.rst:71]:
```suggestion
   install the `language-cylc`_ extension.
```

**Comment by MetRonnie** [comment]
tests now failing pending https://github.com/cylc/cylc-rose/pull/187

**Comment by MetRonnie** [comment]
Latest failure:
```
Warning, treated as error:
None:163: : Spell check: ana: Rose ana analysis modules.
Makefile:59: recipe for target 'spelling' failed
```

**Comment by oliver-sanders** [comment]
Different problem to the broken links. @wxtim could you have a poke at the spelling issue?

**Review by hjoliver — APPROVED** [review]

**Pull request merged**